### DISCUSSION TITLE
[@types/webextension-polyfill]: Wrong typing on sendResponse

### DISCUSSION THREAD

**Discussion by ArthurLobopro** [DefinitelyTyped/DefinitelyTyped#69727] [Issues with a @types package]
The sendResponse callback type has no parameters and it's an error: 

![](https://private-user-images.githubusercontent.com/64273139/335866790-0a70db41-cd14-40ef-8a69-88a13329a8b6.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTczNDMwOTcsIm5iZiI6MTcxNzM0Mjc5NywicGF0aCI6Ii82NDI3MzEzOS8zMzU4NjY3OTAtMGE3MGRiNDEtY2QxNC00MGVmLThhNjktODhhMTMzMjlhOGI2LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA2MDIlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNjAyVDE1Mzk1N1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPThiOTg2YzlhZGRjZDVlNjM2YjRkMThiZjU0M2Q4NGI3ZTVkMjM5MzBhMDRhNjhhOTUzMTkxMDkwMGNlMzI4ZGEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.I3jSNE6e0JmLBe0qCWfsFn2uHIB9t_eIddng9yVPJHA)

Because of that, my code is triggering an error at these lines

![](https://private-user-images.githubusercontent.com/64273139/335866571-ec9126bc-9eab-4d8d-b773-37e37bf71d41.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTczNDMwOTcsIm5iZiI6MTcxNzM0Mjc5NywicGF0aCI6Ii82NDI3MzEzOS8zMzU4NjY1NzEtZWM5MTI2YmMtOWVhYi00ZDhkLWI3NzMtMzdlMzdiZjcxZDQxLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA2MDIlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNjAyVDE1Mzk1N1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTAzN2EyOTIyNGZhZDViNGE1YWUyYmY2YmZiNDdiMTY1YjBjOWFkZDI0M2JjOTc0NGUwMDYyOWE1MzI0NjM0ZDEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.4Rner82_9xBR14Fzja0dmlEMNj1C2FJ1BJ83UmEUrYQ)

[a comment by an automated account is not shown]

**Comment by Lusito** [comment]
Hi @ArthurLobopro, sorry, this somehow slipped past me. There is a fix already in #69704, but it's stuck in review hell :face_with_diagonal_mouth:

**Comment by ArthurLobopro** [comment]
@Lusito no problem my friend, thanks for fixing it!
